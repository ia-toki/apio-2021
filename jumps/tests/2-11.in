209636825ecd2c38299498cd26890ef71dd128c2
200 200
1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 71 72 73 74 75 76 77 78 79 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 96 97 98 99 100 101 102 103 104 105 106 107 108 109 110 111 112 113 114 115 116 117 118 119 120 121 122 123 124 125 126 127 128 129 130 131 132 133 134 135 136 163 138 139 140 141 142 143 144 145 146 147 148 149 150 151 152 153 154 155 156 157 158 159 160 161 162 137 164 165 166 167 168 169 170 171 172 173 174 175 176 177 178 179 180 181 182 183 184 185 186 187 188 189 190 191 192 193 194 195 196 197 198 199 200
0 0 199 199
29 198 199 199
10 13 30 73
36 87 100 145
66 80 109 182
83 86 114 116
0 3 110 141
43 65 132 171
68 92 129 152
19 38 194 195
11 35 90 132
3 133 135 162
52 192 198 198
29 137 184 192
146 166 174 185
30 53 183 186
28 28 119 182
2 5 27 63
49 79 198 199
16 190 197 197
23 53 198 199
53 118 133 167
48 90 168 196
27 27 127 189
28 31 82 89
33 57 158 180
14 20 38 75
25 25 140 185
47 159 175 199
34 81 99 112
118 130 175 195
94 116 150 196
58 77 179 184
15 51 158 171
10 17 80 130
17 108 149 198
99 107 123 194
0 10 14 153
50 78 171 180
29 44 50 160
46 55 114 128
43 146 174 193
6 103 158 175
37 83 144 180
160 193 196 199
40 192 198 198
0 19 36 94
24 41 95 162
115 157 168 178
38 170 186 188
70 139 156 192
41 72 92 145
4 5 175 182
55 123 126 143
47 73 184 193
54 73 115 156
21 38 84 111
81 86 109 153
99 143 170 173
92 139 148 198
42 112 191 193
41 155 181 185
88 190 195 198
37 135 156 177
22 22 121 175
35 56 96 134
138 185 198 199
126 126 145 181
82 142 173 173
0 108 165 197
41 163 198 199
64 72 78 149
53 177 192 193
31 44 171 176
55 111 120 159
11 142 189 199
64 143 154 166
15 33 57 115
124 172 184 191
16 19 65 150
53 63 102 156
9 142 148 157
73 113 163 164
117 129 196 198
35 180 189 195
176 185 199 199
43 140 147 196
22 80 98 172
58 197 198 198
3 142 149 176
141 156 174 181
82 193 194 197
29 37 73 75
52 63 127 187
23 90 153 168
13 106 147 169
123 173 199 199
34 46 51 175
38 54 66 136
127 164 188 192
5 28 166 184
88 132 167 178
26 157 197 199
148 158 198 198
82 131 176 177
62 134 192 192
109 150 179 192
42 42 88 139
82 122 170 183
45 50 146 180
26 77 81 198
108 170 178 196
46 64 160 188
86 182 192 192
70 90 100 198
0 2 176 181
7 74 136 174
1 10 144 156
4 61 188 189
125 192 198 198
34 197 198 199
75 178 181 188
43 46 71 109
39 61 199 199
82 132 147 154
161 179 191 196
173 173 186 192
68 125 135 142
28 160 162 185
20 37 166 182
39 69 111 125
162 169 194 194
59 64 84 90
0 130 134 157
40 61 176 183
2 118 191 198
17 31 63 144
3 31 92 195
50 96 191 196
41 175 186 189
2 18 37 46
7 7 119 127
60 66 182 194
21 46 199 199
44 46 138 158
83 160 195 197
0 0 135 137
105 124 129 135
54 86 183 183
2 6 127 140
152 159 190 194
36 140 194 198
132 179 181 199
24 97 110 112
16 148 176 180
48 98 183 189
40 169 172 174
62 161 169 185
31 72 116 154
14 30 74 76
5 11 131 160
12 41 173 197
163 176 177 187
32 190 198 199
147 171 192 198
47 93 194 198
120 148 155 167
6 148 183 190
12 52 135 187
4 57 166 167
48 130 146 181
25 176 178 197
83 112 140 179
127 143 182 185
10 13 134 178
4 20 37 173
10 191 193 197
21 32 54 168
5 13 45 152
5 22 173 198
17 25 50 110
30 47 84 86
2 26 72 185
77 135 152 170
1 38 107 167
25 28 95 131
122 198 199 199
52 120 143 199
1 5 94 108
5 75 157 171
5 40 134 155
4 12 130 196
0 22 169 173
51 84 120 130
47 168 183 192
114 139 154 167
143 149 151 187
74 76 99 129
58 178 194 199
26 76 179 191
