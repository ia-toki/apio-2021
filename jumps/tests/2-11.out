fc5005cd102fffc504bc1615010ebf88823d65f3
OK
173
1
17
13
29
28
107
67
37
130
55
2
6
22
8
104
91
22
93
7
119
15
52
100
51
80
18
112
13
18
19
21
76
86
63
29
16
4
67
6
59
12
34
54
3
6
17
54
6
16
1
20
144
3
85
42
46
23
8
1
53
19
5
2
99
40
13
11
11
31
35
6
15
101
9
27
1
24
12
46
39
6
24
41
9
14
1
18
1
1
13
1
36
64
47
31
26
5
12
24
112
9
35
37
19
32
17
46
22
87
4
8
73
10
10
148
62
-1
101
6
1
3
25
112
-1
12
13
10
1
103
42
25
20
4
89
47
32
61
69
11
19
112
90
127
-1
33
135
5
71
121
29
32
2
13
14
59
3
7
44
44
120
106
1
8
21
75
1
21
83
83
7
2
25
20
121
17
2
22
32
125
25
37
46
2
69
67
1
17
89
62
94
118
121
36
15
1
2
23
16
77
